declarations/WebpackOptions.d.ts

```diff
@@ -736,6 +736,12 @@ export type AssetParserDataUrlFunction = (
 	source: string | Buffer,
 	context: {filename: string; module: import("../lib/Module")}
 ) => boolean;
+/**
+ * Specifies the convention of exported names.
+ */
+export type CssGeneratorExportsConvention =
+	| ("asIs" | "camelCase" | "camelCaseOnly" | "dashes" | "dashesOnly")
```

Comment:
Let's use `as-is`, `camel-case` and etc, because it was mistake and we decided don't change it, but in webpack we prefer using `-`